internal/cli/iam/organizations/apikeys/accesslists/create.go

```diff
@@ -65,15 +66,52 @@ func (opts *CreateOpts) newWhitelistAPIKeysReq() ([]*atlas.WhitelistAPIKeysReq,
 	return req, nil
 }
 
+func (opts *CreateOpts) newAccessListAPIKeysReq() ([]*atlas.AccessListAPIKeysReq, error) {
+	req := make([]*atlas.AccessListAPIKeysReq, 0, len(opts.ips)+len(opts.cidrs))
+	if len(opts.ips) == 0 && len(opts.cidrs) == 0 {
+		return nil, fmt.Errorf("either --ip or --cidr must be set")
+	}
+	for _, v := range opts.ips {
+		entry := &atlas.AccessListAPIKeysReq{
+			IPAddress: v,
+		}
+		req = append(req, entry)
+	}
+
+	for _, v := range opts.cidrs {
+		entry := &atlas.AccessListAPIKeysReq{
+			CidrBlock: v,
+		}
+		req = append(req, entry)
+	}
+
+	return req, nil
+}
+
 func (opts *CreateOpts) Run() error {
-	req, err := opts.newWhitelistAPIKeysReq()
+	req, err := opts.newAccessListAPIKeysReq()
 	if err != nil {
 		return err
 	}
 
-	r, err := opts.store.CreateOrganizationAPIKeyWhite(opts.ConfigOrgID(), opts.apyKey, req)
+	r, err := opts.store.CreateOrganizationAPIKeyAccessList(opts.ConfigOrgID(), opts.apyKey, req)
 
 	if err != nil {
+		// We keep supporting OM 4.2 and OM 4.4
+		if strings.Contains(err.Error(), "404") {
+			req, e := opts.newWhitelistAPIKeysReq()
+			if e != nil {
+				return e
+			}
+
+			r, e := opts.store.CreateOrganizationAPIKeyAccessListDeprecated(opts.ConfigOrgID(), opts.apyKey, req)
+			if e != nil {
+				return e
+			}
+
+			return opts.Print(r)
+		}
```

Comment:
We should move this to the `store` layer, it should not be the command responsibility to know there are two APIs and how to fallback, I would also use a helper object to pass to the store and let the store handle converting between API types, also moving it there let's us handle the fallback only for Ops Manager

Please let's do this for all commands here